SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMapGeneration.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMapGeneration.h"
+#include "SpatialGDKEditor/Public/SpatialTestSettings.h"
+#include "TestMaps/GeneratedTestMap.h"
+
+DEFINE_LOG_CATEGORY(LogTestMapGeneration);
+
+namespace SpatialGDK
+{
+namespace TestMapGeneration
+{
+bool GenerateTestMaps()
+{
+	UE_LOG(LogTestMapGeneration, Display, TEXT("Deleting the generated test map folder %s."), *UGeneratedTestMap::GetGeneratedMapFolder());
+	if (FPaths::DirectoryExists(UGeneratedTestMap::GetGeneratedMapFolder()))
+	{
+		IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+		const bool bSuccess = PlatformFile.DeleteDirectoryRecursively(*UGeneratedTestMap::GetGeneratedMapFolder());
+		UE_CLOG(!bSuccess, LogTestMapGeneration, Error, TEXT("Failed to delete the generated test map folder %s."),
```

Comment:
Should probably move `bSuccess` to the outer scope to remember if there were any failed steps and return this at the end.
It should probably still be fine to attempt to generate the maps even if deleting the folder failed, though I'm also happy to return early if that fails.

Reply:
Addressed now